subprojects/groovy-yaml/src/spec/doc/yaml-userguide.adoc

```diff
@@ -0,0 +1,112 @@
+//////////////////////////////////////////
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+
+//////////////////////////////////////////
+
+= Processing YAML
+
+Groovy comes with integrated support for converting between Groovy objects and YAML. The classes dedicated to
```

Comment:
I'd tweak the first sentence:
Groovy has an optional `groovy-yaml` module which provides support ... (as before)

Reply:
OK. I will refine the docs according to your suggestions later.